simulation/common/helpers.py

```diff
@@ -554,25 +554,8 @@ def apply_race_timing_constraints(speed_kmh, start_hour, simulation_duration, ra
 
     """
 
-    # (Charge from 7am-9am and 6pm-8pm) for ASC - 13 Hours of Race Day, 9 Hours of Driving
-    # (Charge from 8am-9am and 6pm-8pm) for FSGP
-
-    simulation_hours = np.arange(
-        start_hour, start_hour + simulation_duration / (60 * 60))
-
-    simulation_hours_by_second = np.append(np.repeat(simulation_hours, 3600),
-                                           start_hour + simulation_duration / (60 * 60)).astype(int)
+    not_charge = get_race_timing_constraints_boolean(start_hour, simulation_duration, race_type)
```

Comment:
nit: I would consider renaming to non_changing_array, just because it's not clear that it's representing time